src/layers/conv.jl

```diff
@@ -342,6 +381,7 @@ end
 
 """
     CrossCor(filter, in => out, σ=identity; stride=1, pad=0, dilation=1, [bias, init])
+    CrossCor(weight::AbstractArray, [bias, activation; stride, pad, dilation])
 
 Standard cross convolutional layer. `filter` is a tuple of integers
```

Comment:
```suggestion
Standard cross correlation layer. `filter` is a tuple of integers
```
Is this supposed to say correlation, or is convolution correct?

Reply:
Good catch, it should say correlation.